[2023-01-07T01:57:36.500+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dpt_cosmetic.private_info.check_file_exists scheduled__2023-01-06T00:00:00+00:00 [queued]>
[2023-01-07T01:57:36.639+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dpt_cosmetic.private_info.check_file_exists scheduled__2023-01-06T00:00:00+00:00 [queued]>
[2023-01-07T01:57:36.650+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-07T01:57:36.652+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-07T01:57:36.653+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-07T01:57:36.811+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): private_info.check_file_exists> on 2023-01-06 00:00:00+00:00
[2023-01-07T01:57:36.903+0000] {standard_task_runner.py:55} INFO - Started process 170 to run task
[2023-01-07T01:57:36.928+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dpt_cosmetic', 'private_info.check_file_exists', 'scheduled__2023-01-06T00:00:00+00:00', '--job-id', '488', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp45t6g7d5']
[2023-01-07T01:57:36.970+0000] {standard_task_runner.py:83} INFO - Job 488: Subtask private_info.check_file_exists
[2023-01-07T01:57:37.086+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-01-07T01:57:37.635+0000] {task_command.py:376} INFO - Running <TaskInstance: dpt_cosmetic.private_info.check_file_exists scheduled__2023-01-06T00:00:00+00:00 [running]> on host fdf4acf1d8ed
[2023-01-07T01:57:38.485+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dpt_cosmetic
AIRFLOW_CTX_TASK_ID=private_info.check_file_exists
AIRFLOW_CTX_EXECUTION_DATE=2023-01-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-06T00:00:00+00:00
[2023-01-07T01:57:38.506+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-01-07T01:57:38.510+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 1']
[2023-01-07T01:57:38.668+0000] {subprocess.py:86} INFO - Output:
[2023-01-07T01:57:39.743+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-01-07T01:57:40.197+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dpt_cosmetic, task_id=private_info.check_file_exists, execution_date=20230106T000000, start_date=20230107T015736, end_date=20230107T015740
[2023-01-07T01:57:40.475+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-07T01:57:41.114+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-01-07T05:18:49.398+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dpt_cosmetic.private_info.check_file_exists scheduled__2023-01-06T00:00:00+00:00 [queued]>
[2023-01-07T05:18:49.898+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dpt_cosmetic.private_info.check_file_exists scheduled__2023-01-06T00:00:00+00:00 [queued]>
[2023-01-07T05:18:49.942+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-07T05:18:49.977+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-07T05:18:50.069+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-07T05:18:51.331+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): private_info.check_file_exists> on 2023-01-06 00:00:00+00:00
[2023-01-07T05:18:51.420+0000] {standard_task_runner.py:55} INFO - Started process 483 to run task
[2023-01-07T05:18:51.578+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dpt_cosmetic', 'private_info.check_file_exists', 'scheduled__2023-01-06T00:00:00+00:00', '--job-id', '575', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpfjxwgbmm']
[2023-01-07T05:18:51.671+0000] {standard_task_runner.py:83} INFO - Job 575: Subtask private_info.check_file_exists
[2023-01-07T05:18:51.981+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-01-07T05:18:53.837+0000] {task_command.py:376} INFO - Running <TaskInstance: dpt_cosmetic.private_info.check_file_exists scheduled__2023-01-06T00:00:00+00:00 [running]> on host fdf4acf1d8ed
[2023-01-07T05:18:55.883+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dpt_cosmetic
AIRFLOW_CTX_TASK_ID=private_info.check_file_exists
AIRFLOW_CTX_EXECUTION_DATE=2023-01-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-06T00:00:00+00:00
[2023-01-07T05:18:55.892+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-01-07T05:18:55.901+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 1']
[2023-01-07T05:18:56.250+0000] {subprocess.py:86} INFO - Output:
[2023-01-07T05:18:57.358+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-01-07T05:18:58.111+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dpt_cosmetic, task_id=private_info.check_file_exists, execution_date=20230106T000000, start_date=20230107T051849, end_date=20230107T051858
[2023-01-07T05:18:58.589+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-07T05:18:59.841+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-01-07T05:32:36.853+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dpt_cosmetic.private_info.check_file_exists scheduled__2023-01-06T00:00:00+00:00 [queued]>
[2023-01-07T05:32:37.060+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dpt_cosmetic.private_info.check_file_exists scheduled__2023-01-06T00:00:00+00:00 [queued]>
[2023-01-07T05:32:37.069+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-07T05:32:37.070+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-07T05:32:37.079+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-07T05:32:37.431+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): private_info.check_file_exists> on 2023-01-06 00:00:00+00:00
[2023-01-07T05:32:37.589+0000] {standard_task_runner.py:55} INFO - Started process 141 to run task
[2023-01-07T05:32:37.635+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dpt_cosmetic', 'private_info.check_file_exists', 'scheduled__2023-01-06T00:00:00+00:00', '--job-id', '624', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpkng7d8vi']
[2023-01-07T05:32:37.664+0000] {standard_task_runner.py:83} INFO - Job 624: Subtask private_info.check_file_exists
[2023-01-07T05:32:37.775+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2023-01-07T05:32:38.693+0000] {task_command.py:376} INFO - Running <TaskInstance: dpt_cosmetic.private_info.check_file_exists scheduled__2023-01-06T00:00:00+00:00 [running]> on host fdf4acf1d8ed
[2023-01-07T05:32:39.862+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dpt_cosmetic
AIRFLOW_CTX_TASK_ID=private_info.check_file_exists
AIRFLOW_CTX_EXECUTION_DATE=2023-01-06T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-06T00:00:00+00:00
[2023-01-07T05:32:39.884+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-01-07T05:32:39.894+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 1']
[2023-01-07T05:32:40.072+0000] {subprocess.py:86} INFO - Output:
[2023-01-07T05:32:41.108+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-01-07T05:32:41.504+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=dpt_cosmetic, task_id=private_info.check_file_exists, execution_date=20230106T000000, start_date=20230107T053236, end_date=20230107T053241
[2023-01-07T05:32:42.069+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-07T05:32:43.431+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
